guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,107 @@ ifeval::["{build}" == "satellite"]
 When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the *{project-client-name}* repository.
 endif::[]
 
+[id="creating-hosts-with-uefi-http-boot-provisioning_{context}"]
+=== Creating Hosts with UEFI HTTP Boot Provisioning in IPv4 Network 
```

Comment:
This chapter is about UEFI HTTP Boot in general, it should work with both IPv4 and IPv6.

Reply:
Cool, I was thinking that procedures for IPv4 and IPv6 will be slightly different because of workaround that you need in IPv6, but seems like all bugs in IPv6 will be fixed soon. I will change it to be a general procedure.